features/pull_request.feature

```diff
@@ -921,3 +921,15 @@ BODY
       """
     And the output should match /Given up after retrying for 5\.\d seconds\./
     And a file named ".git/PULLREQ_EDITMSG" should exist
+
+  Scenario: Explicit restrict-maintainer-modify
+    Given the GitHub API server:
+      """
+      post('/repos/mislav/coral/pulls') {
+        assert :maintainer_can_modify => false
```

Comment:
You should also add this assertion to one of the existing scenarios that doesn't use the new flag. The assertion should be that `maintainer_can_modify` was set to either `true` or that it wasn't set at all.